src/features/plan/Plan.js

```diff
@@ -1,70 +1,254 @@
+                                    primary={[
+                                        format(
+                                            parseISO(start_date),
+                                            "dd LLLL yyyy"
+                                        ),
+                                        format(
+                                            parseISO(end_date),
+                                            "dd LLLL yyyy"
+                                        ),
+                                        `${differenceInDays(
+                                            parseISO(end_date),
+                                            parseISO(start_date)
+                                        )} days`,
+                                    ]}
+                                />
+                            </Card>
+                            <Card isNested>
+                                <LightTable
+                                    headers={[
+                                        "Budget",
+                                        "Spent thus far",
+                                        "Remaining",
+                                    ]}
+                                    primary={[
+                                        `$${budget}`,
+                                        `$${totalExpenses}`,
+                                        `$${remaining}`,
+                                    ]}
+                                />
+                            </Card>
                         </Card>
-                        <Card
-                            heading="Expenses"
-                            badge={
-                                <Badge colorScheme="orange">
-                                    Work in Progress
-                                </Badge>
-                            }
-                        >
-                            <ExpenseList />
+                    </GridItem>
+                    <GridItem>
+                        <Card heading="Progress (Time)">
+                            <Card isNested>
+                                <AspectRatio ratio={16 / 9}>
+                                    <CircularProgressbar
+                                        value={progress}
+                                        text={`${progress}% complete`}
+                                        styles={buildStyles({
+                                            pathColor:
+                                                progress > 100
+                                                    ? "#DC2626"
+                                                    : "#3b82f6",
+                                            trailColor: "#f5f5f5",
+                                            textColor:
+                                                progress > 100
+                                                    ? "#DC2626"
+                                                    : "#3b82f6",
```

Comment:
I think you're missing the equality check here `progress > 100` -- otherwise this will never change color. Also, I think green is a more suitable color than red for when the budget has completed as it would be associated with success -- red is fine for exceeding the budget though.